add-on/src/popup/browser-action/tools.js

```diff
@@ -14,6 +14,7 @@ export default function tools ({
   onQuickImport
 }) {
   const activeQuickImport = active && isIpfsOnline && isApiAvailable
+  // TODO(whizzzkid): Remove this once we have a better way to handle this.
   const activeWebUI = active && isIpfsOnline && ipfsNodeType !== 'embedded'
```

Comment:
whole check or just embedded? be more specific in `TODO`, or open an issue. We need to make sure anyone has enough context to take on the TODO at a later point.

ditto for all TODOs added.